extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -34,7 +34,19 @@ class AmpBrightcove extends AMP.BaseElement {
   }
 
   /** @override */
+  buildCallback() {
+    /** @private {?Element} */
+    this.iframe_ = null;
+
+    /** @private {?Promise} */
+    this.iframePromise_ = null;
+  }
+
+  /** @override */
   layoutCallback() {
+    if (this.iframePromise_) {
```

Comment:
Why is this? `layoutCallback` is only allowed to be called once for this element.